contrib/docker/README.md

```diff
@@ -31,9 +14,10 @@ Specify a ``SYNAPSE_CONFIG_PATH``, preferably to a persistent path,
 to use manual configuration. To generate a fresh ``homeserver.yaml``, simply run:
 
 ```
-docker-compose run --rm -e SYNAPSE_SERVER_NAME=my.matrix.host synapse generate
+docker-compose run --rm -e SYNAPSE_SERVER_NAME=my.matrix.host e SYNAPSE_REPORT_STATS=yes synapse generate
```

Comment:
we should not tell people to set `SYNAPSE_REPORT_STATS=yes` without understanding what it does: can you add some words to explain that this is an *example*.